mozilla/foundation.mozilla.org
# updated styling of search bar

Closes #7471 


**Link to sample test home page**: https://foundation-s-7471-resty-znttxo.herokuapp.com/en/privacynotincluded/
**Link to sample test cetogory page**: https://foundation-s-7471-resty-znttxo.herokuapp.com/en/privacynotincluded/categories/health-exercise/

**Steps to test:**

1. Visit one or both of the test links above
2. Note the new style of the search bar
3. Check if behaves as expected
4. If so, testing is complete! 

## Checklist

**Changes in Models:**
